### PR TITLE
Set absolute path for db in blitz-server generator template

### LINKED ISSUE #3926: blitz generate returns "cannot find module db" error when using javascript template
### What is the problem?

When seting up a new project via **blitz new projectname** command with **javascript** as selected language , **blitz generate model testModel** command returns the error **_Error: Cannot find module 'db'_**.

However when selecting typescript as language there is all fine with **blitz generate**.

Am i missing something here?

### Paste all your error logs here:

```
C:\Users\rRene\Coding\manager>blitz generate model project
Loaded env from C:\Users\rRene\Coding\manager\.env.local
Loaded env from C:\Users\rRene\Coding\manager\.env
Error: Cannot find module 'db'
Require stack:
- C:\Users\rRene\Coding\manager\app\blitz-server.js
- C:\Users\rRene\AppData\Roaming\npm\node_modules\blitz\dist\chunks\generate.cjs
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (C:\Users\rRene\Coding\manager\app\blitz-server.js:49:25)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module._compile (C:\Users\rRene\AppData\Roaming\npm\node_modules\blitz\node_modules\esbuild-register\dist\node.js:2258:26)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.newLoader (C:\Users\rRene\AppData\Roaming\npm\node_modules\blitz\node_modules\esbuild-register\dist\node.js:2262:9)
    at Object.extensions..js (C:\Users\rRene\AppData\Roaming\npm\node_modules\blitz\node_modules\esbuild-register\dist\node.js:4801:24) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\rRene\\Coding\\manager\\app\\blitz-server.js',
    'C:\\Users\\rRene\\AppData\\Roaming\\npm\\node_modules\\blitz\\dist\\chunks\\generate.cjs'
  ]
}
```

### Paste all relevant code snippets here:

```
PASTE_HERE (leave the ``` marks)
```

### What are detailed steps to reproduce this?

1. run **blitz new _projectname_**
2. select **javascript** as programming language
3. try to run **blitz generate model _testModel_**
4. returns error "cannot find module 'db'"

### Run `blitz -v` and paste the output here:

```
Blitz version: 2.0.0-beta.14 (global)
Blitz version: 2.0.0-beta.14 (local)
Windows 11 | win32-x64 | Node: v16.13.1


 Package manager: npm

  System:
    OS: Windows 10 10.0.22000
    CPU: (16) x64 AMD Ryzen 7 5800H with Radeon Graphics
    Memory: 18.30 GB / 27.86 GB
  Binaries:
    Node: 16.13.1 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 8.1.2 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    @blitzjs/auth: 2.0.0-beta.14 => 2.0.0-beta.14
    @blitzjs/next: 2.0.0-beta.14 => 2.0.0-beta.14
    @blitzjs/rpc: 2.0.0-beta.14 => 2.0.0-beta.14
    @prisma/client: 4.0.0 => 4.0.0
    blitz: 2.0.0-beta.14 => 2.0.0-beta.14
    next: 12.2.5 => 12.2.5
    prisma: 4.0.0 => 4.0.0
    react: 18.2.0 => 18.2.0
    react-dom: 18.2.0 => 18.2.0
    typescript: ^4.5.3 => 4.8.4
```

### Please include below any other applicable logs and screenshots that show your problem:

_No response_

### PR DESCRIPTION
**Pull request by dillonraphael**
Closes #3926

Since adding custom templates back & esbuild register, when generating a file we require `blitz-server` to read the cliConfig. This was causing issues when using javascript instead of typescript and returned an error `Error: Cannot find module 'db'.` This PR simply sets the path to `../db` instead of `db` and fixes the issue.

### CODE REVIEW

File: packages/generator/src/generator.ts [hunk 1]
```diff
@@ -384,7 +384,7 @@ export abstract class Generator<
       return path.join(
         __dirname,
         "..",
-        process.env.NODE_ENV === "test" ? "./templates" : "./dist/templates",
+        process.env.NODE_ENV === "test" ? "./templates" : "./templates",
```
Comment (beerose) [inline packages/generator/src/generator.ts:387]: Please revert

**Review by beerose — CHANGES_REQUESTED** [review]

File: packages/generator/templates/app/app/blitz-server.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { setupBlitzServer } from '@blitzjs/next';
 import { AuthServerPlugin, PrismaStorage } from '@blitzjs/auth';
 import { simpleRolesIsAuthorized } from '@blitzjs/auth';
 import { BlitzLogger } from 'blitz';
-import db from 'db';
+import db from '../db';
```
Comment (beerose) [inline packages/generator/templates/app/app/blitz-server.ts:5]: I don't understand why this change was needed. It should work out of the box — if it isn't then something is wrong with our paths setup.

**Review by beerose — CHANGES_REQUESTED** [review]

File: packages/generator/templates/app/app/blitz-server.ts (hunk 2 above)
Comment (dillonraphael) [reply packages/generator/templates/app/app/blitz-server.ts:5]: Nothing wrong with the path setup. The issue is with the generator cli command and `esbuild-register`. Can't seem to find db when generating a model in a javascript project (not typescript).

Any advice?

Comment (siddhsuresh) [reply packages/generator/templates/app/app/blitz-server.ts:5]: Hey @dillonraphael, ig this problem can be avoided by removing the dependency on `esbuild` to get the config, instead parsing the file using `recast` and `jscodeshift`. I will open another PR with the changes if this approach is fine.

Comment (dillonraphael) [reply packages/generator/templates/app/app/blitz-server.ts:5]: @siddhsuresh Ya that's an option. Wondering why esbuild-register wouldn't work with compiling js files though.
